index.html

```diff
@@ -0,0 +1,217 @@
+            <button class="btn"><a href="">See Project</a></button>
+        </article>
+
+      </section>
+      <section class="skills">
+        <article class="cards">
+            <figure>
+                <img src="images/art.png" alt="">
+            </figure>
+            <h2 class="title-card">Multi-Post
+                Stories</h2>    
+        <div class="header-details">
+            <h4>CANOPY</h4>           
+            <ul class="dot">              
+                <li>Back End Dev</li>               
+                <li>2015</li>    
+            </ul>
+        </div>
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="skills-card">
+                <li class="skill-1">html</li>
+                <li class="skill-1">css</li>
+                <li class="skill-1">Javascript</li>
+            </ul>
+            <button class="btn"><a href="">See Project</a></button>
+        </article>
+      </section>
+
+      <!-- </* ----------- ABOUT MYSELF ----------- */> -->
+
+      <Section class="about-myself">
+        <article class="about">
+        <h2 class="title-card">About Myself</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a class="link" href="#">LET'S CONNECT</a>
+        <ul class="social-icons">
+           <li><a href=""><img src="images/social-media-icon/tw.png" alt="twitter"></a></li>
+           <li><a href=""><img src="images/social-media-icon/linkedin.png" alt="linkedin"></a></li>
+           <li><a href=""><img src="images/social-media-icon/medium.png" alt="medium"></a></li>
+           <li><a href=""><img src="images/social-media-icon/github.png" alt="github"></a></li>
+           <li><a href=""><img src="images/social-media-icon/v.png" alt="v"></a></li>
+        </ul>
+        <button class="btn"><a href="">Get my resume</a></button>
+        <div class="language">
+        <h3>Languages</h3>
+        <img src="images/browser-icons/chev-dow.svg" alt="">
+        <ul>
+        </div>
+            <li class="skill-2"><img src="images/browser-icons/javascript.svg" alt="">Javascript</li>
+                <li class="skill-2"><img src="images/browser-icons/html.svg" alt="">HTML</li>
+                    <li class="skill-2"><img src="images/browser-icons/css.svg" alt="">CSS</li>
+            
+        </ul>
+        <div class="framwork">
+        <h3>Framework</h3>
+            <img src="images/browser-icons/chev-right.svg" alt="" />
+        </div>
+        <div class="separator-line"></div>
+        <div class="framwork">
+        <h3>Skills</h3>
+        <img src="images/browser-icons/chev-right.svg" alt="" />
+          </button>
+        
+        </div> 
+        <div class="separator-line"></div>
```

Comment:
- [ ] Well done with the `About` section 👍 which is wonderfully made. The alignment of these bocks is great but I think you could do better by making sure they are on the same line 😄 , Kindly fix that

| Your design | Figma template |
| :---: | :---: |
| ![Capture](https://github.com/jerymey/Portfolio-setup/assets/80458199/da958f8b-7b99-4cab-8312-ec132bd6c1ab) | ![Capture1](https://github.com/jerymey/Portfolio-setup/assets/80458199/24683c10-dc19-483c-a04b-569d8e7ecdf7) |